core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -175,7 +175,8 @@ contract Liquidatable is PricelessPositionManager {
     /**
      * @notice Liquidates the sponsor's position if the caller has enough
      * synthetic tokens to retire the position's outstanding tokens.
-     * @dev This method generates an ID that will uniquely identify liquidation for the sponsor.
+     * @dev This method generates an ID that will uniquely identify liquidation for the sponsor. This contract must be
+     * approved to spend at least `tokensLiquidated` of `tokenCurrency` and at least `finalFee` of `collateralCurrency`.
```

Comment:
nit: `finalFee` should be `finalFeeBond`

Reply:
Done.